app/webpacker/css/admin_v3/components/select2.scss

```diff
@@ -229,16 +228,14 @@ label .select2-container {
       display: flex;
       align-items: center;
       justify-content: center;
-      padding-left: 7px;
 
       .select2-search-choice-close {
```

Comment:
stying the close button such that it's displayed with the `icon-remove`. Currently, it's not displayed.

Reply:
Nice to add an improvement while we're here. 
But perhaps there's a reason for this being hidden. I don't think the shop owners have the ability to remove categories. So do the removed ones will get repopulated at the end of the list after saving? (I guess that's what happens when a new category is added).

Testers will probably pick this up, but it's probably worth considering the impact of this a bit more.

Reply:
Thanks for the note @dacook. This PR just tweaks the styling, and the actual functionality and behavior are the same as that of the old UI.
Just to be sure, I validated all the cases of updating the category i.e. add, remove. Both cases are working as expected. When we remove the category, it gets removed as well.
So, I think it's safe to say now the page is exactly mimicking the old UI with new styling. 
Please let me know if you have any questions on this one. Thanks.

Reply:
Apologies, I misunderstood and thought that the remove buttons weren't there before, and that  you were adding them. 
But now I see that they were there already, and you were just fixing the display of them.
![Screenshot 2024-09-17 at 9 19 20 AM](https://github.com/user-attachments/assets/1229d350-e6fa-4557-b10d-d2333822d5a2)